strawberryfields/ops.py

```diff
@@ -1828,8 +1828,11 @@ def __init__(self, V, r=None, decomp=True, hbar=2, tol=1e-6):
             th, self.S = williamson(V, tol=tol)
             self.pure = np.abs(np.linalg.det(V) - (self.hbar/2)**(2*self.ns)) < tol
             self.nbar = np.diag(th)[:self.ns]/self.hbar - 0.5
+            super().__init__([V, r])
+        else:
+            # scale state
+            super().__init__([V*2/hbar, r/np.sqrt(2*hbar)])
```

Comment:
Can this be right? Previously when hbar=2 the tests ran fine, but this becomes `[V, r/2]`, not  `[V, r]` like above.

Reply:
The issue is that the tests aren't testing displacement, see `test_decompositions_integration.py`:

![image](https://user-images.githubusercontent.com/2959003/57143658-e94a0900-6df1-11e9-962a-6d22ad236589.png)

Reply:
This is now fixed @smite